pkg/extender/extender_test.go

```diff
@@ -393,12 +398,17 @@ func pxCSIExtPodDriverTest(t *testing.T) {
 	nodes.Items = append(nodes.Items, *newNode("node3", "node3", "192.168.0.3", "rack1", "", ""))
 	nodes.Items = append(nodes.Items, *newNode("node4", "node4", "192.168.0.4", "rack1", "", ""))
 	nodes.Items = append(nodes.Items, *newNode("node5", "node5", "192.168.0.5", "rack1", "", ""))
+	nodes.Items = append(nodes.Items, *newNode("node6", "node6", "192.168.0.6", "rack1", "", ""))
 
 	if err := driver.CreateCluster(5, nodes); err != nil {
 		t.Fatalf("Error creating cluster: %v", err)
 	}
 	pod := newPod("px-csi-ext-foo", nil)
 
+	if err := driver.UpdateNodeStatus(5, volume.NodeDegraded); err != nil {
```

Comment:
I think we will need to call CreateCluster(6, nodes) above to simulate a 6 node cluster with 1 node in node degraded state.